concent_api/core/message_handlers.py

```diff
@@ -455,30 +468,42 @@ def handle_send_force_payment(request, client_message: message.concents.ForcePay
             reason = message.concents.ForcePaymentRejected.REASON.TimestampError
         )
 
-    # Concent gets list of list of forced payments from payment API where T0 <= payment_ts + PAYMENT_DUE_TIME + PAYMENT_GRACE_PERIOD.
-    list_of_forced_payments = base.get_forced_payments(oldest_payments_ts, requestor_ethereum_public_key, client_public_key, request = request)
+    # Concent gets list of forced payments from payment API where T0 <= payment_ts + PAYMENT_DUE_TIME + PAYMENT_GRACE_PERIOD.
+    list_of_forced_payments = base.get_list_of_payments(
+        concent_rpc             = concent_rpc,
+        requestor_eth_address   = requestor_eth_address,
+        provider_eth_address    = provider_eth_address,
+        payment_ts              = oldest_payments_ts + settings.PAYMENT_DUE_TIME + settings.PAYMENT_GRACE_PERIOD, # Im not sure, check it please
+        current_time            = current_time,
+        to_block                = base.get_number_of_eth_block(concent_rpc),
+        transaction_type        = 'Force',
+    )
 
-    sum_of_payments = base.payment_summary(request = request, subtask_results_accepted_list = client_message.subtask_results_accepted_list, list_of_transactions = list_of_transactions, list_of_forced_payments = list_of_forced_payments)  # pylint: disable=no-value-for-parameter
+    (amount_paid, amount_pending) = base.sum_amount_price_for_provider(
+        list_of_forced_payments         = list_of_forced_payments,
+        list_of_payments                = list_of_transactions,
+        subtask_results_accepted_list   = client_message.subtask_results_accepted_list,
+    )
 
     # Concent defines time T2 (end time) equal to youngest payment_ts from passed SubtaskResultAccepted messages from subtask_results_accepted_list.
     payment_ts = min(subtask_results_accepted.payment_ts for subtask_results_accepted in client_message.subtask_results_accepted_list)
 
-    if sum_of_payments <= 0:
+    if amount_pending <= 0:
         return message.concents.ForcePaymentRejected(
             reason = message.concents.ForcePaymentRejected.REASON.NoUnsettledTasksFound
         )
-    elif sum_of_payments > 0:
-        amount_paid = base.make_payment_to_provider(
-            sum_of_payments,
-            payment_ts,
-            requestor_ethereum_public_key,
-            client_public_key
+    elif amount_pending > 0:
+        tx_hash = base.make_force_payment_to_provider(
```

Comment:
If this value is not used anywhere, don't create variable out of it. I bet that pylint is marking this.